extensions/graphql/README.md

```diff
@@ -3,7 +3,7 @@
 This module provides integration with [GraphQL](https://graphql.org/) using
 [type-graphql](https://typegraphql.com/).
 
-![type-graphql](type-graphql.png)
+![type-graphql](https://raw.githubusercontent.com/strongloop/loopback-next/master/extensions/graphql/type-graphql.png)
```

Comment:
FYI: I fixed the copy-readme.js to ensure image files are copied too but Jekyll still seems to have trouble rendering them.

Reply:
Do you want me to revert this change and you'll fix this image issue?

Reply:
No, the copy didn't help the Jekyll rendering. If you look at https://github.com/strongloop/loopback.io/tree/gh-pages/pages/en/lb4/readmes/loopback-next/extensions/graphql, we have both `.md` and `.png` there. Github web UI displays the image correctly but for some reason, Jekyll does not handle it.